test/e2e/tls_test.go

```diff
@@ -115,14 +118,14 @@ func (data *TestData) checkTLS(t *testing.T, podName string, containerName strin
 	}
 
 	// 2. Set TLSMaxVersion to TLS1.2, then TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256 should be used
-	stdouts = data.opensslConnect(t, podName, containerName, true, apiserver)
+	stdouts = data.opensslConnect(t, podName, containerName, true, apiserver, nodeIPv4, nodeIPv6)
 	for _, stdout := range stdouts {
-		assert.True(t, strings.Contains(stdout, fmt.Sprintf("New, TLSv1.2, Cipher is %s", cipherSuiteStr)),
+		assert.True(t, strings.Contains(stdout, fmt.Sprintf("SSL connection using TLSv1.2 / %s", cipherSuiteStr)),
 			"Cipher Suite used by %s apiserver should be the TLS1.2 one '%s', output: %s", apiserverStr, cipherSuiteStr, stdout)
 	}
 }
 
-func (data *TestData) opensslConnect(t *testing.T, pod string, container string, tls12 bool, port int) []string {
+func (data *TestData) opensslConnect(t *testing.T, pod string, container string, tls12 bool, port int, nodeIPv4, nodeIPv6 string) []string {
```

Comment:
in the context of this function, it makes more sense to call the parameters `dstIPv4` and `dstIPv6` instead of `nodeIPv4` and `nodeIPv6`